src/test/util_tests.cpp

```diff
@@ -156,19 +157,58 @@ BOOST_AUTO_TEST_CASE(util_FormatISO8601Date)
 
 struct TestArgsManager : public ArgsManager
 {
+    std::map<std::string, std::vector<std::string>> m_override_args;
+    std::map<std::string, std::vector<std::string>> m_config_args;
+
     TestArgsManager() { m_network_only_args.clear(); }
     std::map<std::string, std::vector<std::string> >& GetOverrideArgs() { return m_override_args; }
     std::map<std::string, std::vector<std::string> >& GetConfigArgs() { return m_config_args; }
+    bool ParseParameters(int argc, const char* const argv[], std::string& error) {
+        bool result = ArgsManager::ParseParameters(argc, argv, error);
```

Comment:
**const** bool would help here, it actually too a wile to see that it is not changed throughout this function.